vk_requests/session.py

```diff
@@ -46,6 +48,9 @@ def __init__(self, app_id=None, user_login=None, user_password=None,
         self.interactive = interactive
         self._access_token = None
         self._api_version = api_version
+        self._client_secret = client_secret
+        self.two_fa_supported = two_fa_supported
```

Comment:
Any reason why we're not keeping it "private"?
Like this
```suggestion
        self._two_fa_supported = two_fa_supported
```

Reply:
I think this change was not applied